transform/macros/map_class_fp.sql

```diff
@@ -12,10 +12,10 @@
 } -%}
 
 case
-    {% for k, v in class_fips_dict.items() %}
+    {% for k, v in class_fips_dict.items() -%}
```

Comment:
what's this dash here and below for?

Reply:
This trims some whitespace in the rendered file. Entirely cosmetic